codegen/core/src/main/java/software/amazon/smithy/python/codegen/generators/SetupGenerator.java

```diff
@@ -255,4 +262,199 @@ private static void writeReadme(
             writer.popState();
         });
     }
+
+    /**
+     * Write the files required for sphinx doc generation
+     */
+    private static void writeDocsSkeleton(
```

Comment:
We don't need to do this now, but lets add a todo to make this configurable. We should be able to turn off generating all the sphinx setup files and select a theme.